attack_run.sh

```diff
@@ -3,9 +3,18 @@
 set -u
 
 export QUAY_URL=<URL>
+export UUID=$(uuidgen)
+export DURATION=120
 export PARALLELISM=10
-export NUM_USERS=1000
-export RATE=10
+export NUM_USERS=10
+export RATE=40
+export PREFIX=perf-test
+export ES=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
+export ES_PORT=80
+export DB=mysql57
+export TEST_NAME=rook_test
+export QUAY_VERSION=3
```

Comment:
(Nitpick) Want to change the default here to `"3.3.0"` or something along those lines? Soon, we'll have `3.3.1` and the next major release will be `3.4.0`

Reply:
Patch coming.